codegen/src/lib.rs

```diff
@@ -9,17 +9,22 @@
 // except according to those terms.
 
 //! Code generation for [`cucumber_rust`] tests auto-wiring.
+//!
+//! [`cucumber_rust`]: https://docs.rs/cucumber_rust
 
 #![deny(
-    missing_debug_implementations,
     nonstandard_style,
     rust_2018_idioms,
+    rustdoc::broken_intra_doc_links,
+    rustdoc::private_intra_doc_links,
     trivial_casts,
-    trivial_numeric_casts
+    trivial_numeric_casts,
+    unsafe_code
```

Comment:
If this is the case, it's better to do `#![forbid(unsafe_code)]`, because the `deny` can be overwritten with local `allow`, while `forbid` is not.

And place `unsafe: forbidden` badge in README.

Reply:
It's also would be nice to forbid non-ascii idents.